source/install/upgrading-rdo-2.html.adoc

```diff
@@ -201,21 +201,70 @@ If  the `crudini` command is not available, install the `crudini` package:
 ```
 ====
 
-2.  On your Compute host, run:
+.  On your Compute host, run:
 +
 ```
 # systemctl stop '*nova*'
 # yum -d1 -y upgrade \*nova\*
-# su -s /bin/sh -c "nova-manage api_db sync" nova
+```
+
+. In Ocata, the Compute service requires at least two Cells V2 cell databases, the fist database is always called `cell0` and the second database can be named arbitrarily, for example, `cell1`.
++
+Assuming that you have already created a `cell0` database with Cells V1 in your Newton environment, create and map the `cell0` database with Cells V2:
++
+```
+# su -s /bin/sh -c "nova-manage cell_v2 map_cell0" nova
+```
+
+. Populate the `cell0` database by updating the `db` database schema:
++
+```
 # su -s /bin/sh -c "nova-manage db sync" nova
 ```
 
-3.  After you have upgraded all of your hosts, you will want to remove the API limits configured in the previous step. On all of your hosts:
+. Create a `cell1` cell:
++
+```
+# su -s /bin/sh -c "nova-manage --config-file /etc/nova/nova.conf cell_v2 create_cell --name=cell1 --verbose" nova
+```
++
+Run this command for each cell in your environment.
+
+. Discover compute hosts:
++
+```
+# su -s /bin/sh -c "nova-manage cell_v2 discover_hosts --verbose" nova
+```
++
+[NOTE]
+====
+If you add more compute hosts in your environment, remember to run the `discover_hosts` command again.
+====
+
+. List registered cells to get their UUIDs:
++
+```
+# nova-manage cell_v2 list_cells
+```
+
+. Pass the UUIDs to the `map_instances` command to map instances to cells:
++
+```
+# su -s /bin/sh -c "nova-manage cell_v2 map_instances --cell_uuid <cell UUID>" nova
+```
+
+. Update the `api_db` database schema:
++
+```
+# su -s /bin/sh -c "nova-manage api_db sync" nova
+```
+
+.  After you have upgraded all of your hosts, you will want to remove the API limits configured in the previous step. On all of your hosts:
 +
 ```
 # crudini --del /etc/nova/nova.conf upgrade_levels compute
 ```
-4.  Restart the Compute service on all the Controller and Compute nodes:
+.  Restart the Compute service on all the Controller and Compute nodes:
 +
 ```
 # systemctl start openstack-nova-api \
```

Comment:
AFAIK Compute node will have openstack-nova-compute only (not listed). Controller will have everything else.